scripts/lighthouseConfig.js

```diff
@@ -0,0 +1,8 @@
+module.exports = {
```

Comment:
Reason for this: https://github.com/bbc/simorgh/pull/2044#issuecomment-505454752